src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int SECOND_ELEMENT = 1;
+    private static final String TXT_SUPPLY = "supply";
+    private static final String TXT_BUY = "buy";
+    private static final String TXT_RESULT = "result";
+    private static final String TXT_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            File firstFile = new File(fromFileName);
+            File secondFile = new File(toFileName);
+
+            int buy = 0;
+            int supply = 0;
+
+            List<String> stringList = Files.readAllLines(firstFile.toPath());
+            for (String s : stringList) {
+                if (s.contains(TXT_BUY)) {
+                    String[] str = s.split(TXT_SEPARATOR);
+                    buy += Integer.parseInt(str[SECOND_ELEMENT]);
```

Comment:
Here you are checking if the line contains 'buy', splitting the line, and then parsing the integer. This code is duplicated in the next if block for 'supply'. Consider extracting this into a separate method.